Cargo.toml

```diff
@@ -10,9 +10,9 @@ license = "GPL-3.0"
 
 [dependencies]
 num-traits = "0.1.36"
-num-complex= "0.1.36"
+num-complex= { git = "http://github.com/rust-num/num" }
```

Comment:
This must be rewritten. In current version of num-complex (v0.1.36), `Complex64` does not have `#[repr(C)]`, and this causes many warnings since I set `pub use Complex64 as fftw_complex`. This is fixed in the following PR, but is not released:
https://github.com/rust-num/num/pull/275